examples/meta/generator/targets/ruby.json

```diff
@@ -33,6 +33,18 @@
         "LongRealMatrix": "$name = NMatrix.float($arguments)",
         "ComplexMatrix": "$name = NMatrix.scomplex($arguments)"
     },
+    "ListInit": {
+        "CharVector": "$name = NArray['byte', [$arguments])",
+        "ByteVector": "$name = NArray['byte', [$arguments])",
+        "WordVector": "$name = NArray['byte', [$arguments])",
+        "IntVector": "$name = NArray['sint', [$arguments])",
+        "LongIntVector": "$name = NArray['int', [$arguments])",
+        "ULongIntVector": "$name = NArray['int', [$arguments])",
+        "ShortRealVector": "$name = NArray['sfloat', [$arguments]))",
+        "RealVector": "$name = NArray['float', [$arguments])",
+        "LongRealVector": "$name = NArray['float', [$arguments])",
+        "ComplexVector": "NArray['scomplex', [$arguments])"
+    },
```

Comment:
I tested this out with the most recent numo/NArray version

Reply:
ah actually, what type are the zero constructors? I can't find it anywhere

Reply:
what do you mean?

Reply:
sorry, this was when I was confused as to how to construct with a list of args in ruby. was simpler than I thought! :D